src/components/Button/README.md

```diff
@@ -46,11 +48,23 @@ Buttons should:
   confusion and make merchants unsure of what to do next.
 - Be positioned in consistent locations in the interface.
 
+### Buttons versus links
+
+Buttons are used primarily for actions, such as “Add”, “Close”, “Cancel”, or “Save”. Plain buttons, which look similar to links, are used for less important or less commonly used actions, such as "view shipping settings".
+
+Links are used primarily for navigation, and usually appear within or directly following a sentence.
+
+The HTML for buttons and links carries specific meaning. Using these components intentionally and consistently results in:
```

Comment:
Making the first part of this sentence clearly refer to React components could clarify the intent:

```suggestion
The markup resulting from `Button` and `Link` carries meaning. Using these components intentionally and consistently results in:
```

What do you think?